InstantGrinder/InstantGrinder/InstantGrinderPlugin.cs

```diff
@@ -72,27 +99,89 @@ public bool TryGetPlayerByName(string playerName, out MyPlayer player)
             return false;
         }
 
+        private void PlayerRefresh(MyPlayer player)
+        {
+            var playerEndpoint = new Endpoint(player.Id.SteamId, 0);
+            var replicationServer = (MyReplicationServer)MyMultiplayer.ReplicationLayer;
+            object clientData;
+
+            try
+            {
+                clientData = ClientStates.Invoke(replicationServer)[playerEndpoint];
+            }
+            catch
+            {
+                return;
+            }
+
+            List<IMyReplicable> lists = new List<IMyReplicable>(Replicables.Invoke(clientData).Count);
+            foreach (var pair in Replicables.Invoke(clientData))
+            {
+                lists.Add(pair.Key);
+            }
+
+            foreach (var replicable in lists)
+            {
+                RemoveForClient.Invoke(replicationServer, replicable, clientData, true);
+                ForceReplicable.Invoke(replicationServer, replicable, playerEndpoint);
+            }
+            player.Character.GetInventory().Refresh();
+        }
```

Comment:
I briefly remember you saying that refreshing the character inventory didn't work out -- is this code making an effect?